### PR TITLE
Allow external test files.

### PR DESCRIPTION
**Pull request by GeorgeErickson**
Make external test (e.g.  `*_test` packages) work.

### CODE REVIEW

**Comment by GeorgeErickson** [comment]
To make this work for packages that have internal and external tests would require a much bigger refactor. Since you would possible be writing two generated files per package.

File: test_pkg_examples/examples_test.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package pkg_example_test
+
+//go:generate gunit -suffix 1
```
Comment (mdwhatcott) [inline test_pkg_examples/examples_test.go]: I'm not familiar with the 'suffix' flag. What is its purpose?

Comment (GeorgeErickson) [inline test_pkg_examples/examples_test.go]: Nice catch, My previous implementation used a new flag.

**Comment by mdwhatcott** [comment]
So, the merge conflict is my fault and is also easy to resolve. I just have one remaining concern:

What if a developer defines tests in one file that are 'inside' the package and then defines tests in another file that are 'outside' the package. With your changes, all generated tests would be defined in the 'outside' package, which would prevent the tests that were defined in the 'inside' package from compiling. It seems like to really do this right you would have to generate two files, one for the tests defined 'inside' the package (if any) and one file for the tests defined 'outside' the package (if any).

**Comment by mdwhatcott** [comment]
@GeorgeErickson - Since gunit no longer generates test code (we use reflection to enumerate the test cases of a fixture struct) you can now define fixtures internally and externally out-of-the-box.

**Pull request closed without merging**